MAUI/Expander/Getting-Started.md

```diff
@@ -0,0 +1,400 @@
+                                <Label CharacterSpacing="0.25"  FontSize="14" Text="No.8 Blossom St, Washington, DC 20019" Grid.Row="1"/>
+                                <Label CharacterSpacing="0.25"  FontSize="14" Text="(202) 547-3555" Grid.Row="2"/>
+                            </Grid>
+                        </syncfusion:SfExpander.Content>
+                    </syncfusion:SfExpander>
+                </Border>
+            </StackLayout>
+        </ScrollView>
+    </ContentPage.Content>
+</ContentPage>
+
+Now, run the application to render the desired output.
+
+![.Net MAUI Expander](Images/maui-expander-with-gettingstarted.png)
+
+
+## Animation duration
+
+The `SfExpander` allows you to customize the expanding and collapsing duration by using the `AnimationDuration` property. By default, the animation duration is `200 milliseconds`. 
+
+{% tabs %}
+{% highlight xaml %}
+    <syncfusion:SfExpander x:Name="expander" AnimationDuration="250"/>
+{% endhighlight %}
+{% highlight c# %}
+    expander.AnimationDuration = 250;
+{% endhighlight %}
+{% endtabs %}
+
+## Animation easing
+
+The `SfExpander` allows you to customize the rate of change of parameter over time or animation style by using the `AnimationEasing` property. By default, the animation easing is `Linear`. 
+
+{% tabs %}
+{% highlight xaml %}
+         <syncfusion:SfExpander x:Name="expander" AnimationEasing="SinOut"/>       
+{% endhighlight %}
+{% highlight c# %}
+    expander.AnimationEasing = Syncfusion.Maui.Expander.AnimationEasing.SinOut;
+{% endhighlight %}
+{% endtabs %}
+
+## Expand and Collapse 
+
+The `SfExpander` allows you to programmatically expand and collapse by using the `IsExpanded` property of SfExpander. The user can control the expand and collapse interactions by handling the `Expanding` and `Collapsing` events.
+
+{% tabs %}
+{% highlight xaml %}
+    <syncfusion:SfExpander x:Name="expander" IsExpanded="True"/>        
+{% endhighlight %}
+{% highlight c# %}
+    expander.IsExpanded = true;
+{% endhighlight %}
+{% endtabs %}
+
+### Customize the expander when collapsing or expanding the SfExpander
+
+You can customize the expander by using the `Collapsed` event.This event occurs after a `SfExpander` is collapsed when tapping on the header. It provides information related to the `collapsed` event through the `ExpandedAndCollapsedEventArgs`.
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfExpander x:Name ="expander" Collapsed="Expander_Collapsed">
+    <syncfusion:SfExpander.Header>
+        <Grid >
+            <Label x:Name="label"  Text="Veggie burger" FontSize="Large"/>
+        </Grid>
+    </syncfusion:SfExpander.Header>
+	
+    <syncfusion:SfExpander.Content>
+        <Grid>
+            <Label Text="Veggie burger, garden burger, or tofu burger uses a meat analogue, a meat substitute such as tofu, textured vegetable protein, seitan (wheat gluten), Quorn, beans, grains or an assortment of vegetables, which are ground up and formed into patties."/>
```

Comment:
<Label Text="Veggie burger, garden burger, or tofu burger uses a meat analog, a meat substitute such as tofu, textured vegetable protein, seitan (wheat gluten), Quorn, beans, grains or an assortment of vegetables, which are ground up and formed into patties."/>